tensorboard/webapp/widgets/card_fob/card_fob_controller_test.ts

```diff
@@ -383,6 +394,7 @@ describe('card_fob_controller', () => {
     it('moves the start fob based on adapter getStepHigherThanMousePosition when mouse is dragging down and is below fob', () => {
       const fixture = createComponent({
         timeSelection: {start: {step: 1}, end: null},
+        enableRangeSelection: true,
```

Comment:
false? End is null and rangeSelectionEnabled is set to true in many tests...I think they should all be false(or, if it does not particularly matter maybe not there at all and go with default as described above?)

Reply:
Given that I've gone ahead with adding a default value as requested below, I'm just removing the override wherever end is null.